tests/server_unit_test/test_admin.py

```diff
@@ -134,3 +136,89 @@ def test_add_file_no_store_exists(test_client):
     response = AdminRequestFailedResponse.model_validate_json(response.content)
 
     assert response.reason == "Store not_a_store does not exist."
+
+
+def setup_test_file(session, test_orm, full_path):
+    # Create or find a store in the database
+    db_store = (
+        session.query(test_orm.StoreMetadata).filter_by(name="local_store").first()
+    )
+    if not db_store:
+        db_store = test_orm.StoreMetadata(
+            name="local_store", description="A local store for testing"
+        )
+        session.add(db_store)
+        session.commit()
+
+    # Create file and instance records in the database
+    file_data = open(full_path, "rb").read()
+    db_file = test_orm.File.new_file(
+        filename="test_file_to_verify.txt",
+        size=len(file_data),
+        checksum=hashlib.md5(file_data).hexdigest(),
+        uploader="test_uploader",
+        source="test_source",
+    )
+    instance = test_orm.Instance.new_instance(
+        path=str(full_path),
+        file=db_file,
+        store=db_store,
+        deletion_policy="ALLOWED",
+    )
+    session.add_all([db_file, instance])
+    session.commit()
+
+    return db_file, instance
```

Comment:
This code already exists; please don't duplicate it. You should look at the fixtures that we have already defined in the various `conftest.py` files and their usage throughout the test suite.